src/Tizen.WebView/Tizen.WebView/Context.cs

```diff
@@ -94,5 +106,38 @@ public CookieManager GetCookieManager()
             }
             return _cookieManager;
         }
+
+        /// <summary>
+        /// Clears HTTP caches in the local storage and all resources cached in memory.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void ClearResourceCache()
+        {
+            Interop.ChromiumEwk.ewk_context_resource_cache_clear(_handle);
+        }
+
+        /// <summary>
+        /// Notify low memory to free unused memory.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void NotifyLowMemory()
+        {
+            Interop.ChromiumEwk.ewk_context_notify_low_memory(_handle);
+        }
+
+        /// <summary>
+        /// Sets the download callback.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetDownloadCallback(IntPtr downloadUserData, SetStartDownloadCallback startDownloadCb)
```

Comment:
Why not provide as `Event`?

```suggestion
        public event EventHandler<DownloadStartedEventArgs> DownloadStarted;
```

Reply:
Chromium-efl can have only one callback for download, but EventHandler can be added or removed.
So, could we keep the current implementation?

Reply:
@BasavarajPS , `IntPtr downloadUserData` is not useful, plz remove it.
And then we can call `Interop.ChromiumEwk.ewk_context_did_start_download_callback_set` with `IntPtr.Zero`.

Reply:
Done

Reply:
> 
> 
> Chromium-efl can have only one callback for download, but EventHandler can be added or removed.
> So, could we keep the current implementation?

No, We can only on callback into native API. but we can dispatch a event to multiple receiver on C# layer

Reply:
``` c#
 DownloadStartedCallback onDownload;

 public event EventHandler<DownloadStartedEventArgs> DownloadStarted;

// on constructor
 onDownload = (url) => {
     DownloadStarted?.Invoke(this, new DownloadStartedEventArgs(url));
 };
 SetDownloadCallback(onDownload); 
```

Reply:
I means that  we want to have only one callback for download.
The purpose of this callback is to delegate content download job to application side.
Content download should be processed only one time, but in your code, developer can subscribe `Downloadstarted` many times.

Reply:
You said, it invoked to delegate content download.
but you do not provide way to content download
So, i think it is a just event to notify start of download.

could you explain how to download content with url?

and it should be renamed to `SetDownloadRequestDelegate`

Reply:
In the legacy native app, developer can request to download content using 'app control' with `APP_CONTROL_OPERATION_DOWNLOAD`. This operation requires only URL.
AFAIK, TizenFx also supports 'app control'.
We can rename method as 'SetDownloadRequestDelegate'.

Reply:
I have more question, if sever is require a some of cookie value, ex) to check authentication. how to pass current session infomation into app control?

Reply:
Currently, there is no way that we can add additional information of download request into 'app control'.
But I think that we can extend argument if 'app control' supports it, because this method is internal.

Reply:
Done